src/serializationStrategies/XpathRangeSelector/xpath.ts

```diff
@@ -118,10 +119,10 @@ export function getXPathForElement(targetElement: Node, offset: number, referenc
     let search: Node | null = focus.childNodes[offset - 1];
 
     while (search) {
-      if (isTextOrTextHighlight(search)) {
+      if (isTextOrTextHighlight(search) && !isScreenReaderNode(search)) {
```

Comment:
i feel like `isTextOrTextHighlight` should not return true for a screen reader node, it isn't a highlight, i guess its a part of the highlight, but does it really need the highlight attribute on it?

Reply:
Removing it is causing not loading some highlights
before:
![image](https://user-images.githubusercontent.com/31478212/110499463-8cf45680-80f8-11eb-9ec0-db2a00274db9.png)

after:
![image](https://user-images.githubusercontent.com/31478212/110499402-79e18680-80f8-11eb-818e-d995fd97a68c.png)

When I modified `isHighlight` to return `false` for screen reader nodes then 16 tests failed :D 

I feel like this is currently working because of this logic to omit highlights when finding the xpath and if we remove the highlight attribute from screenreader nodes we will have to add the same logic for these nodes.

Do you want me to debug this deeper?

Reply:
I've added a commit https://github.com/openstax/highlighter/pull/42/commits/3de38600e3704d61ad6482ec225f5a4107797afe that changed bunch of functions' names in `xpath.ts` and removed `data-highlighter` from screen reader nodes